README.md

```diff
@@ -68,20 +68,47 @@ operating systems)
 See [Parameters](#parameters) for the list of properties. You will be prompted for any property you didn't include in
 the file if it is needed for a given stage.
 
+## High-level overview of each step
+
+01 - Load the original data into a temporary database for processing (EF and AA only)
```

Comment:
You need a blank line between each step, right now GitHub is display them in a single paragraph.